### ISSUE TITLE
Atlas list of OT candidates for curation

### ISSUE THREAD

**Issue by ElaineMcA** [opentargets/platform#47]
@ckongEbi commented on [Tue Jan 16 2018](https://github.com/opentargets/data_release/issues/59)

Hi Laura,

   Thanks for your time going through your curation process!  As discussed will you be able collate the list of OT related candidates in your format (or those recently curated) into a single google docs spreadsheet? We could then look at the page knowing, which experiments are curated and to be expected from the atlas dump.

In parallel, I will check with the related parties to get a list of disease types of interest/focus for early next year.

Have a good holidays and see you next year !

CK

---

@iandunham commented on [Tue Jan 16 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-358004989)

Hi
Just to add into this, there was a request to get the date from this paper curated (PMID: 28892060). I think there is a bunch of expression data in there from IBD, but could you have a look with Laura?

---

@ckongEbi commented on [Tue Jan 16 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-358006675)

@iandunham  Yes the plan is for her to maintain the OT candidate list sheet and we will add it in there. Including this IBD papers. 

Are there particular disease area or disease branch which is of our focus/priority ? From the discussion, it speeds up her curation when she focus on a particular disease for a while, instead of going randomly. 

---

@ckongEbi commented on [Wed Mar 07 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-371118541)

Disease list from sent to Laura for focus curation
GSK: Respiratory system disease and autoimmune diseases.
Biogen:  AD,PD,MS,SMA,ALS,stroke,ophthalmology,pain
Takeda: To be added

---

@ckongEbi commented on [Tue Mar 20 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-374638220)

On 20/03/2018 15:21, Chuang Kee Ong wrote:
>
> Hi Laura,
>
>     Thanks for your help looking at the initial curation list, we are hoping for something similar to what GWAS has...
>
> https://docs.google.com/spreadsheets/d/1gMOMwbQnQVYloAkwgTlNKUbWrWROJersh_iO5tVhtEY/edit#gid=499408605
> Cheers,
> CK
>
> On 07/03/2018 12:06, Chuang Kee Ong wrote:

---

@ckongEbi commented on [Mon Apr 16 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-381612979)

Additional curation request to Atlas. 

As pointed out by Stuart at GSK, there is a meta-analysis published by AstraZeneca with some reference to GEO studies.

https://www.nature.com/articles/s41598-017-10930-w

GSE4845213, GSE6126014, GSE8963215, GSE5904516, GSE4954117, GSE1565318 and GSE3381419. 

 

Anna Mae Diehl did some of the earliest work

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3982589/

---

@ckongEbi commented on [Fri May 18 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-390205941)

Create a spreadsheet for curation list
https://drive.google.com/open?id=1HOC5msCHfzbbO2oCPG1JtI9FNLuRj0qHsIfCa9BXFCo

---

@ckongEbi commented on [Mon Jun 11 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-396190541)

With regards to miRNA experiments, Laura identified ~400 studies in ArrayExpress that might be of OT interest.
[ArrayExpress-Experiments-180426-121455.txt](https://github.com/opentargets/data_release/files/2089482/ArrayExpress-Experiments-180426-121455.txt)

They need to process the required studies through atlas for us to get it via their data feed. The [iRAP](https://github.com/nunofonseca/irap) pipeline is used to process the RNA-seq data and needed to be tweak to process miRNA data. I am testing their pipeline and feedback to Nuno for changes to achieve this. 

---

@ElaineMcA commented on [Tue Aug 14 2018](https://github.com/opentargets/data_release/issues/59#issuecomment-412912209)

Curation queue updated by Laura and OT request process shared also.  Regular check-in will be put on calendar for queues generally as part of new data request process.

**Comment by MichaelaEBI** [comment]
This is now in hand with the spreadsheet and a separate ticket for miRNA data has been opened: [Issue 229](https://github.com/opentargets/platform/issues/229)
